src/modules/collection/utils.ts

```diff
@@ -48,7 +48,7 @@ export function getExplorerURL(collection: Collection, chainId: ChainId) {
 
   // We're replacing org and hardcoding zone here because it only works on that domain for now, to avoid adding new env vars
```

Comment:
```suggestion
  // We're replacing org and hardcoding zone here because it only works on that domain for now, to avoid adding new env vars. Also, we use `ropsten` for the NETWORK because it is the only working network for .zone
```